src/Pact/Analyze/Remote/Server.hs

```diff
@@ -122,7 +122,7 @@ moduleNotFoundP = MP.string "<interactive>:"
                *> digitsP *> MP.char ':'
                *> digitsP *> MP.char ':'
                *> MP.string " Module \""
-               *> fmap (ModuleName . T.pack) (MP.some $ MP.notChar '"')
+               *> fmap (\a -> ModuleName (T.pack a) Nothing) (MP.some $ MP.notChar '"')
```

Comment:
This is something I'd like the folks handling the remote stuff to take a look at

Reply:
Done here: #350

Reply:
This is actually breaking a test I think - we'll discuss tomorrow